### PR TITLE
feat(CardLoader): new component

### PR DESCRIPTION
**Pull request by ZhenYongSeekAsia**
** Please provide as much detail as possible, but feel free to remove irrelevant sections **

Introduce generic card loader, to enable job details team create a job preview loader
http://seek-asia-style-guide--b6ad3fb777198b18641a1564ac76352c263eab74.surge.sh/CardLoader

Note: 
Changing the gridValues in  grid.js from lessToJs to manual copy grom grid.less because the !!raw-loader!./grid.less will fail the unit test. 

BREAKING CHANGE:

{{ Description of breaking API change }}

RFC URL:

{{ URL of associated RFC for API change }}

MIGRATION GUIDE:

Before:

```js
{{ Old code }}
```

After:

```js
{{ New code }}
```

EXAMPLE USAGE:

```js
{{ Code }}
```

### CODE REVIEW

**Comment by mengtzu** [comment]
http://seek-asia-style-guide--b6ad3fb777198b18641a1564ac76352c263eab74.surge.sh/CardLoader

File: theme/layout/grid.js [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
-const gridValues = lessToJs(grid);
+//based on grid.less
+const gridValues = {
+  '@grid-base': '4px',
+  '@grid-row-height': '4px',
+  '@grid-gutter-width': '@grid-base * 4',
+  '@grid-column-width': '60px',
+  '@grid-container-width': '940px',
+  '@gutter-width': '20px'
+}
```
Comment (hscgavin) [inline theme/layout/grid.js]: ☝️ copy & paste? we should look into `'!!raw-loader!./grid.less'` and figure out why it's broken?

File: theme/layout/grid.js [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
 export default gridValues;
+
+export const GRID_BASE = 4;
```
Comment (hscgavin) [inline theme/layout/grid.js]: How about `GRID_BASE = parseInt(gridValues['@grid-base'], 10);`?

File: theme/layout/grid.js [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+export const getMultipliedGridLength = (multiplier = 1) => {
+  return `${(GRID_BASE * multiplier)}px`
```
Comment (hscgavin) [inline theme/layout/grid.js]: Eslint is not watching you `;`

File: theme/layout/grid.js (hunk 1 above)
Comment (ZhenYongSeekAsia) [reply theme/layout/grid.js]: @hscgavin i have run out of idea to fix it. 😂 would you like to help me out?

**Pull request merged**